### ISSUE TITLE
Running DaVinci on the grid: PythonOptsCmakeParserError

### ISSUE THREAD

**Issue by weissercn** [lhcb/first-analysis-steps#232]
Hey guys,

I am following the tutorial and on the "Running DaVinci on the grid" chapter we are told to enter this line:
j.application.readInputData('data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py')

I get the following error:

ERROR    PythonOptsCmakeParserError:
[Errno 2] No such file or directory: 'data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py'
ERROR    Error: ApplicationConfigurationError: Unable to parse the job options. Please check options files and extraopts.

**Comment by alexpearce** [comment]
This string:

```
data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py
```

represents the path to a file. In the directory you're running Ganga, Ganga will look for a directory called `data`, and will try to load the file inside that directory called `MC_2012_….py`.

It looks like Ganga can't find that file. Did you download it? Instructions on how to get that file are given in a previous step.

**Comment by weissercn** [comment]
That does make sense. Just for completion it might be helpful to specify that the file should be saved to a folder called 'data'. 

I was just confused about the difference between data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST and data/MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py.  
When I renamed the file to add the '.py' it worked.

**Comment by alexpearce** [comment]
Yes, the file downloaded from the DIRAC portal isn't given the `.py` extension in some browsers. We could add a callout box to say that.

And I think you're right that we could be clearer that the file should live in a `data/` directory (or don't have the directory at all).
